### PR TITLE
removed m.room.message.feedback

### PR DESCRIPTION
**Pull request by thomaslantern**
Signed-off-by: Thomas Scott <shikisha@hotmail.com>

MSC: https://github.com/matrix-org/matrix-spec-proposals/pull/3582

Preview: https://pr1035--matrix-spec-previews.netlify.app

### CODE REVIEW

**Review by turt2live — CHANGES_REQUESTED** [review]
Thanks! This just needs a changelog entry for "Removed `m.room.message.feedback` per MSC0000." (whatever that MSC is).

**Comment by thomaslantern** [comment]
> Thanks! This just needs a changelog entry for "Removed `m.room.message.feedback` per [MSC0000](https://github.com/matrix-org/matrix-spec-proposals/issues/0000)." (whatever that MSC is).

Changelog entry added, thanks!

File: changelogs/client_server/newsfragments/3582.clarification [hunk 1]
```diff
@@ -0,0 +1 @@
+Removed m.room.message.feedback per MSC3582.
```
Comment (turt2live) [inline changelogs/client_server/newsfragments/3582.clarification:1]:
```suggestion
Removed `m.room.message.feedback` per [MSC3582](https://github.com/matrix-org/matrix-spec-proposals/pull/3582).
```

**Review by turt2live — APPROVED** [review]
Thanks!

**Pull request merged**